packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -71,9 +71,7 @@ class _Details extends CoreElement {
 
   void update(TimelineEvent event) {
     attribute('hidden', false);
-    _duration.text = 'Duration:'
-        ' ${_microsAsMsText(event.duration)} - '
-        '[${event.startTime} - ${event.endTime}]';
```

Comment:
the times in nanoseconds don't seem very human readable. I'm happy to be convinced otherwise but I would suggest removing these until we find a human readable form. For example, time in ms relative to the start of the frame might be useful to show.

Reply:
sgtm.